FordLabs/PeopleMover
# Fix eslint errors & add eslint to pre-commit hook

### What was done:
- [x] Ran `npm run lint-fix` to fix any eslint errors that are auto-fixable
- [x] Manually fixed remaining eslint errors (not warnings)
- [x] Added eslint to the husky pre-commit hook so eslint errors are caught before pushing code

### How to test:
1. Run `npm run lint` and ensure that there are no more errors (not warnings) remaining.
2. Run `npm run lint-fix` and ensure that that command does not make any more changes.
3. Attempt to make a commit and ensure that the pre-commit hook runs eslint and that it passes successfully.
